api/server/routes/submodel.py

```diff
@@ -6,11 +6,10 @@
 from server.services.submodel_service import SubmodelService
 from basyx import ObjectStore
 
-from server.utils.pagination import Pagination
-from server.utils.decorator import limited
+from server.utils.decorator import paginated
 
 
-class SubmodelRouter(Pagination):
+class SubmodelRouter():
```

Comment:
```suggestion
class SubmodelRouter:
```

I think it's considered better style to omit redundant parantheses.